### PR TITLE
Restructure code with configuration token implementation 

### PR DESCRIPTION
**Pull request by TarasMaliuchenko**

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#! /usr/bin/env sh
+echo " .:|:.:|:. "
+echo " C I S C O "
+echo "  SecureX "
+echo
+echo " Development Dockerfile build script."
+echo
+
+module_name="Cyberprotect"
```
Comment (AlinaBlahun) [inline build.sh:9]: It used to be "Threatscore | Cyberprotect" here https://github.com/CiscoSecurity/tr-05-serverless-cyberprotect/blob/develop/module_settings.json , I am not sure what option is right

File: module_type.json.sample [hunk 2]
```diff
@@ -0,0 +1,57 @@
+{
+  "title": "Threatscore | Cyberprotect",
+  "default_name": "Threatscore | Cyberprotect",
+  "short_description": "Threatscore gives a computed score about a level of threat for any (known) observables.",
+  "description": "Threatscore by Cyberprotect is based on NICKI - New Intelligence for Cyber-threat Knowledge for Inter-connectivity.\n\n\n- Every day, we process a little over a billion events.\n\n\n- We rely on a team of experts who are first and foremost cybersecurity enthusiasts.\n\n\n- Our knowledge of cyber threats is based on the high quality and diversity of brains and information we have in our lab.\n\n\n- Our engine has the ability to inter-connect with a wide range of equipment. This makes it possible to offer everyone the opportunity to optimize their cybersecurity in an efficient and unrestricted way.",
+  "tips": "When configuring this integration, you must first add the Threatscore Cyberprotect Module\n\n1. Complete the **Add New Threatscore Cyberprotect Module** form:\n    - **Module Name** - Leave the default name or enter a name that is meaningful to you.\n2. Click **Save** to complete the Threatscore Cyberprotect module configuration",
```
Comment (AlinaBlahun) [inline module_type.json.sample:6]: Missing `Entities Limit`

Comment (TarasMaliuchenko) [reply module_type.json.sample]: Fixed

File: build.sh (hunk 1 above)
Comment (TarasMaliuchenko) [reply build.sh]: Fixed

**Review by AlinaBlahun — APPROVED** [review]

**Review by AnastasiiaRozlyvan — APPROVED** [review]

**Pull request merged**